app/Models/Context.php

```diff
@@ -9,27 +9,47 @@ final class FreshRSS_Context {
 	/**
 	 * @var FreshRSS_UserConfiguration|null
 	 */
-	public static $user_conf = null;
+	public static $user_conf;
 
 	/**
 	 * @var FreshRSS_SystemConfiguration|null
 	 */
-	public static $system_conf = null;
-
+	public static $system_conf;
+	/**
+	 * @var array<FreshRSS_Category>
+	 */
 	public static $categories = array();
+	/**
+	 * @var array<string>
+	 */
 	public static $tags = array();
-
+	/**
+	 * @var string
+	 */
 	public static $name = '';
+	/**
+	 * @var string
+	 */
 	public static $description = '';
-
+	/**
+	 * @var int
+	 */
 	public static $total_unread = 0;
+	/**
+	 * @var array<string, int>
+	 */
 	public static $total_starred = array(
 		'all' => 0,
 		'read' => 0,
 		'unread' => 0,
 	);
-
+	/**
+	 * @var int
+	 */
 	public static $get_unread = 0;
+	/**
+	 * @var array<string, bool>
```

Comment:
Wrong type, the values can also be integers. Fix coming

Reply:
https://github.com/FreshRSS/FreshRSS/pull/5064/commits/194911b263098d91764f92bfc9c7a9a4843a67ef